opennms-doc/guide-admin/src/asciidoc/text/snmp/snmp-encryption.adoc

```diff
@@ -0,0 +1,17 @@
+
+=== Encrypting SNMP Credentials
+
+SNMP credentials like readCommunity/WriteCommunity, authPassPhrase/privPassPhrase can be saved on the file system in encrypted format.
+
+==== Enabling encrypting on SNMP Config
+
+Specifying a key for encryption would enable encryption on SNMP config.
+[source, sh]
+----
+echo 'org.opennms.snmp.encryption.key=abc12345' >> "$OPENNMS_HOME/etc/opennms.properties.d/snmp-encryption.properties"
```

Comment:
Requiring the key set via a system property and storing it directly on the file-system negates the benefit subsequently storing it in SCV.

Reply:
Yeah, not storing it directly in SCV.  They key is digested to a password and then that is used to encrypt and decrypt password.
Both key/password are stored in SCV.

Reply:
Got it. Is there any need to expose this key then? Seems like we could keep this fixed and then expose a simple boolean on/off instead.

Reply:
yeah, we could keep it fixed since the digested password will always be different for each application and key never changes in the life time of application.